components/interactive-example/with-choices.js

```diff
@@ -104,23 +111,29 @@ export const InteractiveExampleWithChoices = (Base) =>
     }
 
     #render() {
+      const id = randomIdString();
+
       return html`
-        <div class="template-choices">
+        <div class="template-choices" aria-labelledby=${id}>
           <header>
-            <h4>${decode(this.name)}</h4>
-            <mdn-button id="reset" @click=${this._reset} variant="secondary"
+            <h4 id=${id}>${decode(this.name)}</h4>
+            <mdn-button
+              id="reset"
+              @click=${this._reset}
+              variant="secondary"
+              .disabled=${!this.__choiceUpdated}
               >${this.l10n`Reset`}</mdn-button
             >
           </header>
-          <div
+          <ul
             class="choice-wrapper"
             @click=${this.#choiceFocus}
             @focus=${this.#choiceSelect}
             @update=${this.#choiceUpdate}
```

Comment:
Address https://github.com/mdn/fred/pull/542/files/dcde79fd3a4fee22dba096ce5144aef32c610fbd#r2273743349

```suggestion
            @update=${this.#choiceUpdate}
            aria-label=${this.l10n`Value select`}
```